src/GovukNotify/Interfaces/IBaseClient.cs

```diff
@@ -1,16 +1,17 @@
 ﻿using System;
 using System.Net.Http;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Notify.Interfaces
 {
     public interface IBaseClient
     {
-        Task<string> GET(string url);
+        Task<string> GET(string url, CancellationToken cancellationToken);
 
-        Task<string> POST(string url, string json);
+        Task<string> POST(string url, string json, CancellationToken cancellationToken);
 
-        Task<string> MakeRequest(string url, HttpMethod method, HttpContent content = null);
+        Task<string> MakeRequest(string url, HttpMethod method, CancellationToken cancellationToken, HttpContent content = null);
```

Comment:
To make this more of a non breaking change can you make these  cancellation tokens default to null and move them to the last param